examples/colab-notebook-processing_data_using_SQl_and_local_database/README.md

```diff
@@ -0,0 +1,3 @@
+# Colab Notebook for processing data using SQl and local database
+This folder contians the tutorial for <a href="https://github.com/vmware/versatile-data-kit/wiki/Processing-data-using-SQL-and-local-database"> Processing data using SQL and local database</a> example.
+Link to the Google Colab Notebook - <a href="https://colab.research.google.com/github/Indranil0603/versatile-data-kit/blob/Indranil%2FIndranil0603%2FColab-notebook-processing-data-using-SQL-and-local-database/examples/colab-notebook-processing_data_using_SQl_and_local_database/colab-notebook-processing_data_using_SQl_and_local_database.ipynb">Google Colab</a>
```

Comment:
I don't think we need to name the notebook so long? Just `sqlite-example-notebook.ipynb` seems fine ?